spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SchemaConverters.java

```diff
@@ -408,6 +399,35 @@ private DataType getStandardDataType(Field field) {
     }
   }
 
+  @VisibleForTesting
+  static DecimalType createDecimalTypeFromNumericField(Field field) {
```

Comment:
can we create the same check if the SQL type is `BIGNUMERIC` as well? https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/741bdb33bf5c06fb3fad9defffa65ea1470d2690/spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SchemaConverters.java#L375